<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPW</class>
 <widget class="QMainWindow" name="ResetPW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>University Faclilities Management System (UFMS)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="display0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_studid">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="continue_btn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>320</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CONTINUE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_StudentID">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>230</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PLEASE INSERT STUDENT ID :</string>
     </property>
    </widget>
    <widget class="Line" name="oframe2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>781</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="oframe4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>781</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="oframe3">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>10</y>
       <width>20</width>
       <height>581</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="copyright">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>570</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>© SLAUATRON 2019.</string>
     </property>
    </widget>
    <widget class="Line" name="oframe1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>20</width>
       <height>581</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rback2login">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>16</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BACK TO LOGIN PAGE</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_lbl">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>292</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Student ID not found. Please try again.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="display1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_answer">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="continue_btn_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>320</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CONTINUE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_StudentIDlbl">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>STUDENT ID:</string>
     </property>
    </widget>
    <widget class="Line" name="oframe2_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>781</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="oframe4_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>781</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="oframe3_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>10</y>
       <width>20</width>
       <height>581</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="copyright_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>570</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>© SLAUATRON 2019.</string>
     </property>
    </widget>
    <widget class="Line" name="oframe1_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>20</width>
       <height>581</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rback2login_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>16</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BACK TO LOGIN PAGE</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_lbl_2">
     <property name="geometry">
      <rect>
       <x>298</x>
       <y>293</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Answer incorrect. Please try again.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_currentstudid">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>studentid</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_seq">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>192</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SEQURITY QUESTION:</string>
     </property>
    </widget>
    <widget class="QLabel" name="seques_lbl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>461</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>seques_lbl</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ans">
     <property name="geometry">
      <rect>
       <x>248</x>
       <y>262</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ANSWER:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="display2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>358</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_changepw">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>320</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CHANGE PASSWORD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_StudentIDlbl_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>STUDENT ID:</string>
     </property>
    </widget>
    <widget class="Line" name="oframe2_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>781</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="oframe4_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>781</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="oframe3_4">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>10</y>
       <width>20</width>
       <height>581</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="copyright_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>570</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>© SLAUATRON 2019.</string>
     </property>
    </widget>
    <widget class="Line" name="oframe1_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>20</width>
       <height>581</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rback2login_3">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>16</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BACK TO LOGIN PAGE</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_lbl_3">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>292</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>New password cannot be the same as old password!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_currentstudid_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>studentid</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ans_2">
     <property name="geometry">
      <rect>
       <x>218</x>
       <y>261</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NEW PASSWORD:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="db_lbl">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>570</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected to UFMS!</string>
    </property>
   </widget>
   <zorder>display2</zorder>
   <zorder>db_lbl</zorder>
   <zorder>display0</zorder>
   <zorder>display1</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit_studid</sender>
   <signal>returnPressed()</signal>
   <receiver>continue_btn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_answer</sender>
   <signal>returnPressed()</signal>
   <receiver>continue_btn_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_password</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_changepw</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
